theano/gpuarray/opt.py

```diff
@@ -342,9 +345,6 @@ def apply(self, fgraph):
                 nb = change_tracker.nb_imported
```

Comment:
Remove the change tracker completly, it can't help you. It only track change to an existing fgraph and here we don't do inplace change to an fgraph, we build an independent one in parallel.